### PR TITLE
doc: edit for clarity

### PR DESCRIPTION
**Pull request by RandyMcMillan**

### CODE REVIEW

**Comment by jonasschnelli** [comment]
meh ACK f454f16d23d9a4799450faa46ce63803f26d86c1

File: depends/README.md [hunk 1]
```diff
@@ -100,10 +100,19 @@ options will be passed to bitcoin's configure. In this case, `--disable-wallet`.
...
+
+    make download-osx # fetch all sources needed for macOS builds
+
+For Windows:
+
+    make download-win # fetch all sources needed for win builds
+
+For Linux:
+
+    make download-linux # to fetch all sources needed for linux builds
```
Comment (hebasto) [inline depends/README.md:115]: It makes it more confusing, IMO.
What does "For macOS" mean: a host system or a target one?

Let's leave it as is :)

**Comment by hebasto** [comment]
@MarcoFalke Why is this labeled "Needs gitian build" ?

**Comment by MarcoFalke** [comment]
I am testing something and this pull has been selected as the victim

File: depends/README.md (hunk 1 above)
Comment (jonatack) [reply depends/README.md:115]: I agree with @hebasto that this is less clear than the way it is currently.

**Comment by fanquake** [comment]
Thanks, however I agree with the other comments. I'm not sure that the additional titles & whitespace are clarifying anything; the existing doc seems pretty clear as is.

**Comment by RandyMcMillan** [comment]
Besides clarity...it is easier to copy and paste the command. The change also conforms with the existing indented code block markdown as opposed to the fenced code block markdown.
Indented code blocks are more portable across markdown interpreters and should probably be a standard practice for markdown in the documentation. Fenced code blocks with syntax highlighting should be avoided all together.

**Comment by MarcoFalke** [comment]
I think it would have helped to state the motivation in the commit message or pull request description. Sometimes it is hard to see the reasoning from just a diff.

**Pull request closed without merging**